packages/strapi-hook-knex/lib/index.js

```diff
@@ -137,9 +142,8 @@ module.exports = strapi => {
             break;
           case 'sqlite3':
             // Create the directory if it does not exist.
-            const directory = path.dirname(path.resolve(strapi.config.appPath, options.connection.filename));
-            if (!fs.existsSync(directory)){
-              fs.mkdirSync(directory);
+            if (!fs.existsSync(fileDirectory)){
```

Comment:
This function is deprecated (see https://nodejs.org/api/fs.html#fs_fs_exists_path_callback). You should use `fs.access` instead.

Reply:
I know that we often use such functions in the codebase to check whether or not a file or a folder does exactly exist like in here https://github.com/strapi/strapi/blob/39f4d8884f5d963dd00454b04e19f17cce9f2f4d/packages/strapi-plugin-documentation/services/Documentation.js#L35-L40

@Aurelsicoko I think that not only we @pierreburgy should change this function and put it the strapi-utils...